src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -213,6 +213,23 @@
  * <pre>
  * &lt;module name=&quot;WhitespaceAround&quot;/&gt;
  * </pre>
+ * <p>Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public Test(){} // violation
+ *     public static void main(String[] args) {
+ *         if (foo) { // ok
+ *             // body
+ *         }
+ *         else{ // violation
+ *             // body
+ *         }
+ *         for (int i = 1; i &gt; 1; i++) {} // violation
```

Comment:
please make it `// 2 violations , space after '{' is expected`
there same for line below

Reply:
Done @romani